docker-compose.test.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  sut: 
+    #build: .
+    image: 5899797fc3e8
+    #image: ppiper/mta-archive-builder
+    volumes:
+      - ./test/goss:/goss-test
+    working_dir: /goss-test
+
+    command:
+      - /bin/sh
+      - -c
+      - |
+          mkdir .goss
+          curl -fsSL https://goss.rocks/install | GOSS_DST=.goss sh
```

Comment:
💡 use shared named volumes to do the curl fetch in another container to avoid curl installation in build artefact.